code/modules/jobs/job_types/chief_medical_officer.dm

```diff
@@ -26,6 +26,16 @@
 	display_order = JOB_DISPLAY_ORDER_CHIEF_MEDICAL_OFFICER
 	bounty_types = CIV_JOB_MED
 
+	mail_goodies = list(
+		/obj/effect/spawner/lootdrop/organ_spawner = 10,
+		/obj/effect/spawner/lootdrop/memeorgans = 4,
+		/obj/item/scalpel/alien = 1,
+		/obj/item/hemostat/alien = 1,
+		/obj/item/retractor/alien = 1,
+		/obj/item/circular_saw/alien = 1,
+		/obj/item/surgicaldrill/alien = 1,
+		/obj/item/cautery/alien = 1
```

Comment:
note that these can be deconstructed for alien tech

I'd suggest adding abductor glands to this pool at a pretty high rate; they were one of the best parts about pubbystation (rip)